### ISSUE TITLE
Incorrect Relationships for Test Case File "The containment can be indirect 2/2"

### ISSUE THREAD

**Issue by giuseppeverduciALMA** [buildingSMART/IDS#388]
The `.ifc` file for the test case **"The containment can be indirect 2/2"** (`partof/fail-the_containment_can_be_indirect_2_2.ifc`) contains errors in the relationships:

- **Line `#3`:** `IFCRELAGGREGATES` -> `IFCRELCONTAINEDINSPATIALSTRUCTURE`
- **Line `#5`:** `IFCRELCONTAINEDINSPATIALSTRUCTURE` -> `IFCRELAGGREGATES`

For both relationships, the last two attributes need to be swapped.

Additionally, it is necessary to verify the associated `.ids` file (`relation` attribute of `partOf`) to ensure it aligns correctly with the intent.

**Comment by Moult** [comment]
Agree, the test is wrong, mistake made in this commit https://github.com/buildingSMART/IDS/commit/3345939befc669dacbcc5be27ed2888ec3a579fc

**Comment by CBenghi** [comment]
@giuseppeverduciALMA is it easy for you to write a PR?

**Comment by giuseppeverduciALMA** [comment]
> [@giuseppeverduciALMA](https://github.com/giuseppeverduciALMA) is it easy for you to write a PR?

Hi @CBenghi, I just opened the PR for this issue: #407
